source/_integrations/binary_sensor.mqtt.markdown

```diff
@@ -114,7 +114,8 @@ device:
       required: false
       type: string
 device_class:
-  description: Sets the [class of the device](/integrations/binary_sensor/#device-class), changing the device state and icon that is displayed on the frontend.
+  description: Sets the [class of the device](/integrations/binary_sensor/#device-class), changing the device state and icon that is displayed on the frontend. The `device_class` can be `None`.
```

Comment:
Unlike Python, `None` is not a valid value in YAML to set. It would be either `null` or `~`.